src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/AdditionalFactorsForWomenTest.kt

```diff
@@ -29,7 +29,12 @@ class AdditionalFactorsForWomenTest {
   private val clock = Clock.fixed(LocalDateTime.of(2020, 1, 1, 0, 0).toInstant(ZoneOffset.UTC), ZoneId.systemDefault())
   private val communityApiClient: CommunityApiClient = mockk(relaxUnitFun = true)
   private val assessmentApiService: AssessmentApiService = mockk(relaxUnitFun = true)
-  private val additionalFactorsForWomen: AdditionalFactorsForWomen = AdditionalFactorsForWomen(clock, communityApiClient, assessmentApiService)
+  private val additionalFactorsForWomen: AdditionalFactorsForWomen = AdditionalFactorsForWomen(
+    clock,
+    communityApiClient,
+    assessmentApiService,
+    CommunityApiService(communityApiClient)
```

Comment:
Lazy I know but soon this will no longer have the client so I can move the method then